docs/static/ea-integration-tutorial.asciidoc

```diff
@@ -0,0 +1,185 @@
+[[ea-integrations-tutorial]]
+== Using {ls} with Elastic {integrations} (Beta) tutorial
+
+
+Logstash elastic-integration Filter
+Plugin Guide
+Ingest node pipelines in Logstash
+
+
+Process overview
+* Configure Fleet to send data from Elastic Agent to Logstash
+* Create an Elastic Agent policy with the necessary integrations
+* Configure Logstash to use the elastic_integration filter plugin
+
+
+Logstash elastic-integration Filter Plugin Guide
+
+Overview
+The purpose of this guide is to walk through the steps necessary to configure Logstash to transform events
+collected by the Elastic Agent using our pre-built ingest node pipelines that normalize data to the Elastic
+Common Schema. This is possible with a new beta feature in Logstash known as the elastic-integration
+filter plugin.
+Using this new plugin, Logstash reads certain field values generated by the Elastic Agent that tells Logstash to
+fetch pipeline definitions from an Elasticsearch cluster which Logstash can then use to process events before
```

Comment:
```suggestion
Using this new plugin, Logstash reads certain field values generated by the Elastic Agent, and uses them to apply the transformations from Elastic Integrations so that it can further process events before 
```